src/shared/sam/cli/samCliBuild.ts

```diff
@@ -75,7 +79,14 @@ export class SamCliBuildInvocation {
         private readonly args: SamCliBuildInvocationArguments,
         private readonly context: { file: FileFunctions } = { file: getDefaultFileFunctions() }
     ) {
-        this.args.invoker = this.args.invoker ?? new DefaultSamCliProcessInvoker()
+        this.args.invoker =
+            this.args.invoker ??
+            new DefaultSamCliProcessInvoker({
+                preloadedConfig: new DefaultSamCliConfiguration(
```

Comment:
why doesn't `DefaultSamCliProcessInvoker` set this by default? Would save a few imports and this duplicated code in at least 2 files (`samCliBuild.ts` and `localLambdaRunner.ts`).